src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -366,83 +366,83 @@ class VM_Version : public Abstract_VM_Version {
    */
   enum Feature_Flag {
 #define CPU_FEATURE_FLAGS(decl) \
-    decl(CX8,               "cx8",               0)  /*  next bits are from cpuid 1 (EDX) */ \
-    decl(CMOV,              "cmov",              1)  \
-    decl(FXSR,              "fxsr",              2)  \
-    decl(HT,                "ht",                3)  \
+    decl(CX8,               cx8,               0)  /*  next bits are from cpuid 1 (EDX) */ \
+    decl(CMOV,              cmov,              1)  \
+    decl(FXSR,              fxsr,              2)  \
+    decl(HT,                ht,                3)  \
                                                      \
```

Comment:
Please, revert alignment change to reduce size of changes. We can do it in mainline later.

Reply:
Okay, I missed that. Keep XSTR. Why aarch64 code does not use XSTR?